src/components/Other/NavBarIcons.js

```diff
@@ -0,0 +1,225 @@
+import React, {useState, useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+
+import withStyles from '@material-ui/core/styles/withStyles';
+import Button from '@material-ui/core/Button';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import PersonIcon from '@material-ui/icons/Person';
+import NavBarDropDownMenu from '../../../src/components/Other/NavBarDropDownMenu';
+
+const NavBarIcons = ({logout, classes}) => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [openUser, setOpenUser] = useState(false);
+  const [openNotification, setOpenNotification] = useState(false);
```

Comment:
So, there's the opportunity here to use a single variable here rather than two booleans, e.g. to indicate which menu is open. This will scale better if we want to add a third icon as well.

Reply:
Done. Thanks.

Reply:
I refactored a lot of code. You can take a look anytime if you want to review again.